gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapper.java

```diff
@@ -317,13 +318,9 @@ private Optional<Condition.ConditionClinicalStatus> buildClinicalStatus(CD links
     }
 
     private CodeableConcept generateCategory() {
-        Coding coding = new Coding();
-        coding.setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-ConditionCategory-1");
-        coding.setCode("problem-list-item");
-        coding.setDisplay("Problem List Item");
-
-        return new CodeableConcept()
-            .addCoding(coding);
+        return createCodeableConcept("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-ConditionCategory-1",
+                                     "problem-list-item",
+                                     "Problem List Item");
```

Comment:
I actually find the original code more readable, it's immediately clear what values each field has. With this new representation, I can't see what each of these strings is being populated into for the coding.

Reply:
I will add JavaDocs to improve readability...

Reply:
I added JavaDocs to make code more readable and working with it easier.